### PR TITLE
feat: accordion

### PR DESCRIPTION
**Pull request by trishaprile**
[![PR App][icn]][demo] | Fix RM-9808
:-------------------:|:----------:

## 🧰 Changes
- Implement an mdx built-in component for `<Accordion />`!

image

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

**Review by kellyjosephprice — APPROVED** [review]
LGTM, but it looks like this will need to be updated for dark mode?

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.78</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.78)
<br>

File: components/Accordion/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const Accordion = ({ children, title }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleToggle = () => {
+    setIsActive(!isActive);
+  }
+  
+  return (
+    <div className="Accordion">
```
Comment (rafegoldberg) [inline components/Accordion/index.tsx:13]: This should probably be marked up as an actual `<details>` element, or at least set [the appropriate `[aria-role]` for the wrapper and summary sections](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details)! CC: @trishaprile

File: components/Accordion/style.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    display: flex;
+    font-size: 14px;
+    justify-content: space-between;
+    padding: 10px;
+    position: relative;
+    text-align: left;
+    width: 100%;
+  }
+
+  .icon {
```
Comment (rafegoldberg) [inline components/Accordion/style.scss:22]: @trishaprile—per [our convo on #948](https://github.com/readmeio/markdown/pull/948#discussion_r1710381622) can we also update these classes to follow our BEM selector conventions?

**Pull request merged**